src/witan/send/vis/general_population.clj

```diff
@@ -56,6 +61,8 @@
                          :serie-base-def base-gp-serie-def
                          :colors-and-points (merge (wsc/domain-colors-and-points domain-key historical-data)
                                                    {99 {:color [152.0 223.0 138.0 255.0], :point \V}})
+                         ;; 99 is used as a placeholder where data structures require a number be present but is
```

Comment:
add `FIXME: ` at the front as I search for that and some modes highlight that for you automagically